app/src/test/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandlerTest.kt

```diff
@@ -41,14 +40,14 @@ class ChangeLanguageEffectHandlerTest {
   fun `the current selected language must be fetched when the load current selected effect is received`() {
     // given
     val selectedLanguage = SystemDefaultLanguage
-    whenever(settingsRepository.getCurrentSelectedLanguage()).thenReturn(Single.just(selectedLanguage))
+    whenever(settingsRepository.getCurrentLanguage()).thenReturn(Single.just(selectedLanguage))
 
     // when
-    effectsSubject.onNext(LoadCurrentSelectedLanguageEffect)
+    effectsSubject.onNext(LoadCurrentLanguageEffect)
 
     // then
     testObserver
-        .assertValue(CurrentSelectedLanguageLoadedEvent(selectedLanguage))
+        .assertValue(CurrentLanguageLoadedEvent(selectedLanguage))
```

Comment:
Can we use something like this for the effect handler tests,
https://github.com/ragunathjawahar/bench-press/blob/mobius-exercise/app/src/test/java/io/redgreen/benchpress/test/EffectHandlerTestCase.kt

```
effectsSubject.onNext(...)

testObserver
    .assert*()
```

exposes too much information?

Reply:
Looks good, I added this!